packages/fxa-payments-server/src/routes/Product/index.tsx

```diff
@@ -38,8 +40,8 @@ export type ProductProps = {
   createSubscriptionStatus: CreateSubscriptionFetchState;
   plansByProductId: (id: string) => Array<Plan>;
   createSubscription: Function;
-  resetCreateSubscription: () => void;
-  resetCreateSubscriptionError: () => void;
+  resetCreateSubscription: any;
```

Comment:
FWIW, using an `any` type is okay during development - but we should work to get more specific with types.

One thing you could do here is use `typeof resetCreateSubscription` to pull in the exact type of the action creator expected here, now that we can actually see that type. This will probably cause issues that need fixing elsewhere that rely on a more relaxed type, though.